en/development/debugging.rst

```diff
@@ -51,8 +51,13 @@ Using the Debugger Class
 
 .. php:class:: Debugger
 
-To use the debugger, first ensure that ``Configure::read('debug')`` is
-set to ``true``.
+To use the debugger, in your app/Config.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.
```

Comment:
There is no such file. I am against making this too concrete. Other files like bootstrap could also set this. No need to overspecify.

Reply:
adjusted. I changed the names. Thank you